lib/gcloud.rb

```diff
@@ -313,4 +313,39 @@ def search scope: nil
     require "gcloud/search"
     Gcloud.search @project, @keyfile, scope: scope
   end
+
+  ##
+  # Creates a new object for connecting to the Logging service.
+  # Each call creates a new connection.
+  #
+  # @param [String, Array<String>] scope The OAuth 2.0 scopes controlling the
+  #   set of resources and operations that the connection can access. See [Using
+  #   OAuth 2.0 to Access Google
+  #   APIs](https://developers.google.com/identity/protocols/OAuth2).
+  #
+  #   The default scope is:
+  #
+  #   * `https://www.googleapis.com/auth/logging.admin`
+  #
+  # @return [Gcloud::Logging::Project]
+  #
+  # @example
+  #   require "gcloud"
+  #
+  #   gcloud = Gcloud.new
+  #   logging = gcloud.logging
+  #   topic = logging.topic "my-topic"
```

Comment:
Ugh, I thought I caught all of those. Updated and removed. These code examples will need to be revisited once we get more of the logging service implemented.